openstudiocore/resources/osversion/1_10_4/OpenStudio.idd

```diff
@@ -8276,9 +8276,14 @@ OS:AirLoopHVAC,
        \type object-list
        \required-field
        \object-list ConnectionNames
-  A14; \field Return Air Bypass Flow Temperature Setpoint Schedule Name
+  A14, \field Return Air Bypass Flow Temperature Setpoint Schedule Name
```

Comment:
Adding fields that are not in the E+ Idd to OSM requires some discussion, @kbenne should definitely weigh in.  How will this get translated to E+?